ufficio_soci/forms.py

```diff
@@ -412,13 +412,20 @@ def clean_data_versamento(self):
 
 class ModuloFiltraEmissioneTesserini(forms.Form):
 
+    def __init__(self, *args, **kwargs):
+        sedi = kwargs.pop("sedi")
+        super(ModuloFiltraEmissioneTesserini, self).__init__(*args, **kwargs)
+        self.fields['sedi'] = forms.ModelMultipleChoiceField(queryset=sedi, initial=sedi)
```

Comment:
Ottimo :)

Solo alcuni commenti:
- Se un argomento e' obbligatorio, includilo esplicitamente nella firma, in modo che non sia opzionale, es.:
  ```python
  def __init__(self, sedi, *args, **kwargs):
  ```
  Cosi' puoi rimuovere la riga:
  ```python
  sedi = kwargs.pop("sedi")
  ```
- Non sostituire il campo gia' presente, impostane semplicemente il queryset (e il valore iniziale). Vedi la documentazione relativa alla pagina [Form Fields: Forms which handle relationships](https://docs.djangoproject.com/en/2.0/ref/forms/fields/#fields-which-handle-relationships).
  ```python
  self.fields['sedi'].initial = sedi
  self.fields['sedi'].queryset = sedi
  ```
  Piuttosto che:
  ```python
  self.fields['sedi'] = forms.ModelMultipleChoiceField(queryset=sedi, initial=sedi)
  ```